### PR TITLE
add all in one solar notebook

### PR DESCRIPTION
**Pull request by JuHyung-Son**
업스테이지 api 다 들어있는 노트북이예요

### CODE REVIEW

File: solar.ipynb [hunk 1]
```diff
@@ -0,0 +1,1090 @@
+{
```
[the remark this answers is not in the document]
Comment (JuHyung-Son) [reply solar.ipynb:1]: commit fixed and removed key from console

**Comment by chosh0615** [comment]
Can we include OCR and KIE features as well?

**Comment by JuHyung-Son** [comment]
@chosh0615 They are already included.
image

**Comment by JuHyung-Son** [comment]
@chosh0615 more user friendly version

on colab.
image

**Review by chosh0615 — APPROVED** [review]
```
                                                                      ▒▒                        
                                                                  ░░░░░░                        
                                                                  ▒▒▒▒                          
                                                                  ▒▒          ▒▒                
                                                            ░░                                  
                                                          ░░▒▒      ░░                          
                                                          ▒▒▒▒░░            ░░                  
                                                        ▒▒▒▒▒▒                                  
                                                        ▒▒▒▒            ░░░░                    
                                                          ▒▒            ▒▒▒▒                    
                                                      ▒▒▒▒            ▒▒  ▒▒                    
                                                    ▒▒▒▒▒▒░░                                    
                                                    ▒▒▒▒▒▒▒▒        ▒▒▒▒▒▒                      
                          ▒▒                        ▒▒▒▒▒▒          ▒▒▒▒▒▒                      
                        ░░▒▒░░                  ██████████████      ▒▒▒▒                        
                        ░░▒▒░░                  ██▒▒▒▒▒▒▒▒▒▒██      ▒▒                          
                      ░░  ▒▒  ░░                ██▒▒▒▒▒▒▒▒▒▒██  ██████████                      
                      ░░  ▒▒  ░░                ██▓▓▓▓▓▓▓▓▓▓██  ██▒▒▒▒▒▒██                      
                    ░░  ░░▒▒    ░░              ██▓▓▓▓▓▓▓▓▓▓██  ██▒▒▒▒▒▒██                      
                    ░░  ░░▒▒░░  ░░              ██▓▓▓▓▓▓▓▓▓▓██  ██▓▓▓▓▓▓██                      
                  ░░  ░░  ▒▒░░    ░░            ██▓▓▓▓▓▓▓▓▓▓██  ██▓▓▓▓▓▓██      ▒▒              
                ░░    ░░  ▒▒  ░░    ░░          ██▓▓▓▓▓▓▓▓▓▓██  ██▓▓▓▓▓▓██    ░░▒▒░░            
                ░░  ░░    ▒▒  ░░                ██▓▓▒▒▓▓▒▒▓▓██  ██▓▓▓▓▓▓██    ░░▒▒░░            
              ░░    ░░  ██▒▒████░░████░░▒▒████████▒▒▓▓▒▒██▒▒██  ██▓▓▓▓▓▓██  ░░  ▒▒  ░░          
            ░░    ░░  ██░░▒▒░░▒▒░░▒▒▒▒▒▒░░░░▒▒▒▒  ▒▒░░▒▒▒▒▓▓██  ██▓▓▓▓▓▓██░░    ▒▒  ░░          
          ░░      ░░  ██░░▒▒░░▒▒▒▒░░▒▒▒▒▒▒░░▒▒▒▒░░▒▒░░▒▒██▓▓██  ██▓▓▓▓▓▓██  ████████  ░░        
        ░░      ░░  ██▒▒▒▒▒▒▒▒▒▒▒▒░░▒▒▒▒▒▒▒▒░░▒▒▒▒▒▒▒▒▒▒████████████████████▒▒▒▒▒▒▒▒██  ░░      
      ░░        ░░  ██████▒▒████████░░████████░░████████▒▒░░▒▒▒▒░░▒▒▒▒░░▒▒▒▒░░▒▒░░▒▒▒▒██  ░░    
████████      ░░    ██░░░░▒▒░░░░░░▒▒░░▒▒▒▒░░░░▒▒░░▒▒▒▒░░▒▒░░▒▒▒▒░░▒▒▒▒░░▒▒▒▒▒▒▒▒▒▒▒▒████████████
██░░░░░░████████    ██▒▒░░▒▒░░▒▒░░▒▒▒▒░░▒▒░░░░▒▒▒▒░░▒▒░░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒████████████░░░░░░░░░░██
██░░░░░░░░░░░░░░██████████████▒▒▒▒▒▒▒▒▒▒░░▒▒▒▒▒▒▒▒▒▒░░▒▒▒▒██████████████░░░░░░░░░░░░░░░░░░░░░░██
  ██░░░░░░░░░░░░▒▒░░░░░░░░░░▒▒████████████████████████████░░░░░░░░░░░░▒▒░░░░░░░░░░░░░░░░░░▒▒▒▒██
  ██░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░▒▒▒▒▒▒▓▓▓▓██
  ██░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░▒▒▒▒▒▒▒▒▓▓▓▓▓▓▓▓██  
    ██░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▓▓▓▓▓▓▓▓██  
    ██░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░▒▒▒▒▒▒▒▒▒▒▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓██  
    ██▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░░░░░░░░░░░░░░░░░░░░░░▒▒▒▒▒▒▒▒▒▒▒▒▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓████░░░░
      ██▒▒▒▒▒▒▒▒▒▒▒▒▓▓▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓████  ██      ░░
      ██▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓████████          ░░░░░░
      ██▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓██    ██░░          ░░░░░░░░░░  
        ██▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓████████████              ░░░░░░░░░░░░░░    
      ░░██▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓██████    ██████░░            ░░░░░░  ░░░░░░░░░░░░░░░░      
  ░░░░░░  ▓▓▓▓▓▓▓▓▓▓▓▓████      ████                        ░░░░░░░░░░░░░░░░░░░░░░░░░░          
  ░░░░          ████                          ░░░░░░    ░░░░░░░░░░░░░░░░░░░░░░░░                
░░░░░░                    ░░░░  ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░                        
░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░                                
  ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░                                            

```

**Review by nayeon-upstage — APPROVED** [review]
LGTM 👍

**Pull request merged**